packages/shared/styles/components/molecules/SfAddToCart.scss

```diff
@@ -3,21 +3,35 @@
 .sf-add-to-cart {
   display: flex;
   &__button {
-    flex: var(--add-to-cart-button-flex, 1);
+    @include font(
```

Comment:
@justyna-13 in this case you change just font-weight than you should use `--button-font-weight: var(--font-semibold);`